api/src/main/java/org/openmrs/Cohort.java

```diff
@@ -35,9 +36,11 @@
 	private String description;
 	
 	private Set<Integer> memberIds;
```

Comment:
We decided to keep them around as utility methods to allow external optimized code that doesn't wish to deal with heavier patient objects